services/sqs/src/it/java/software/amazon/awssdk/services/sqs/SurfaceAreaReview.java

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.sqs;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.core.interceptor.Context;
+import software.amazon.awssdk.core.interceptor.ExecutionAttributes;
+import software.amazon.awssdk.core.interceptor.ExecutionInterceptor;
+import software.amazon.awssdk.services.sqs.batchmanager.SqsAsyncBatchManager;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityBatchRequest;
+import software.amazon.awssdk.services.sqs.model.ChangeMessageVisibilityResponse;
+import software.amazon.awssdk.services.sqs.model.CreateQueueResponse;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageResponse;
+import software.amazon.awssdk.services.sqs.model.Message;
+import software.amazon.awssdk.services.sqs.model.ReceiveMessageRequest;
+import software.amazon.awssdk.services.sqs.model.ReceiveMessageResponse;
+import software.amazon.awssdk.services.sqs.model.SendMessageResponse;
+
+public class SurfaceAreaReview {
+
+
+    private static SqsAsyncClient globalSqsAsyncClient = SqsAsyncClient.create();
+    private static String QUEUE_URL = null;
+    ;
+
+
+
+    @Test
+    void creatingUsingAsyncClient() throws Exception {
+        if (QUEUE_URL == null) {
+            createTestQueue();
+        }
+        ApiCaptureInterceptor apiCaptureInterceptor = new ApiCaptureInterceptor();
+        SqsAsyncClient sqsAsyncClient =
+            SqsAsyncClient.builder().overrideConfiguration(o -> o.addExecutionInterceptor(apiCaptureInterceptor)).build();
+
+
+        //[REVIEW AREA  for batchManager from SqsAsyncClient]
+        SqsAsyncBatchManager sqsAsyncBatchManager = sqsAsyncClient.batchManager();
```

Comment:
Just to double check, sync client doesn't have batchManager right?

Reply:
What happens if users close the batchmanager, would it close the underlying sqs client (if not provided by customers)?